css/index.css

```diff
@@ -773,4 +772,210 @@ body {
+  font-size: 28px; 
+  font-weight: var(--font-weight-boldX);
+  line-height: 24px;
+  letter-spacing: 0.02em;
+}
+
+.cart-summary__svg{
+  fill: var(--text-color);
+}
+
+.cart-summary__content {
+  padding: 30px;
+}
+
+.cart-summary__item{
+  margin-bottom: 15px;
+}
+
+.cart-summary__description {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 10px;
+}
+
+.cart-summary__name,
+.cart-summary__prics{
+  font-weight: var(--font-weight-boldX);
+  font-size: 24px;
+  line-height: 32px;
+  color: var(--text-color);
+}
+
+.cart-summary__discount {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
+  margin-top: 15px;
+}
+
+.cart-summary__discount-description,
+.cart-summary__discount-price{
+  font-weight: var(--font-weight-boldX);
+  font-size: var(--sixteen-pixeles);
+  line-height: var(--eighteen-pixel);
+  color: var(--color-black-medium);
+}
+
+.cart-summary__remove-dish{
+  font-weight: var(--font-weight-boldX);
+  font-size: var(--font-size-smalls);
+  line-height: var(--twenty-pixel);
+  color: var(--color-black-medium);
+  text-decoration: none;
+}
+
+.card-summary__add {
+  background: var(--button-color);
+  color: var(--color-white);
+  text-align: right;
+  border: none;
+  padding: 10px 20px;
+  border-radius: 8px;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  margin-bottom: 30px;
+  font-weight: var(--font-weight-boldX);
+  font-size: 12px;
+  line-height: 14px;
+  margin-left: auto;
+}
+
+.card-summary__bord{
+  width: 500px;
+  border-bottom: 3px var(--color-whiteX);
+  margin-left: -29px;
+}
+
+.card-summary__add::before {
+  content: '+';
+  margin-right: 8px;
+}
+
+.card-summary__order {
+  margin-bottom: 60px;
+}
+
+.cart-summary__text {
+  color: var(--text-color);
+  font-weight: var(--font-weight-boldX);
+  font-size: var(--twenty-pixel);
+  line-height: 24px;
+  letter-spacing: 0.02em;
+  margin-top: 20px;
+  margin-bottom: 15px;
+}
+
+.cart-summary__row {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 10px;
+  align-items: center;
+}
+
+.card-summary__taxes,
+.card-summary__subtotal{
+  font-weight: var(--font-weight);
+  font-size: var(--sixteen-pixeles);
+  line-height: 21px;
+  color: var(--color-black-medium);
+}
+
+.card-summary__price{
+  font-weight: var(--font-weight-bold);
+  font-size: var(--sixteen-pixeles);
+  line-height: 30px;
+  letter-spacing: 0.01em;
+  color: var(--color-black-medium);
+}
+
+.card-summary__price--total{
+  font-weight: var(--font-weight-boldX);
+  line-height: 44px;
+}
+
+.card-summary__request{
+  font-weight: var(--font-weight-bold);
+  font-size: var(--sixteen-pixeles);
+  line-height: 21px;
+  color: var(--color-black-medium);
+}
+
+.cart-summary__go-pay {
+  margin-top: 30px;
+  background: var(--button-color);
+  color: var(--color-white);
+  border: none;
+  padding: 20px;
+  border-radius: 8px;
+  cursor: pointer;
+  margin-bottom: 30px;
+  display: flex;
+  gap: 197px;
```

Comment:
Cuando tiene mas cifras se ve mal el boton. Quizas sirva meterle este codigo y reducir el valor del `gap`. Es `Autumn Cobb Salad`

<img width="486" alt="image" src="https://github.com/user-attachments/assets/1cbf795c-47b3-4d7a-8d9e-5679ea492382">

```suggestion
  gap: 80px; /* puede ser cualquiera */
  width: 100%;
  justify-content: space-between;
```